tablo/interfaces/arcgis/views.py

```diff
@@ -212,13 +212,15 @@ def handle_request(self, request, **kwargs):
 
 
 class QueryView(FeatureLayerView):
-    query_limit_default = 50000
+    query_limit_default = 1000
```

Comment:
I moved `10000` into a module level constant called `QUERY_LIMIT`, and reference it in all three views: for the two you mention I put it directly in the dict, and for `QueryView` I use it as the value for `query_limit_default`.